deployment/deploy.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - hosts: all
-  sudo: yes
+  become: yes
+  become_method: sudo
```

Comment:
As far as I know there parameters requires a newer Ansible version, in this case, it would be at least useful to know, what Ansible version was using you or what minimal Ansible version is required.

Since Ansible evolves quite fast, some times I add this check:
https://github.com/python-dirbtuves/akl.lt/blob/master/deployment/deploy.yml#L14-L15